hover.com/extract.js

```diff
@@ -0,0 +1,29 @@
+// CONFIG BEGIN
+var defaultTTL = 14400;
+// CONFIG END
+
+var table = document.getElementsByClassName('table-body')[0];
+var rows = table.getElementsByClassName('table-row');
+var i, len, row;
+var hostname, type, priority, ttl, destination;
+var output = '';
+
+output += '$TTL ' + defaultTTL + '\n'; // start with default TTL
+
+// skip header and last two rows (add new entry, delete all entries)
```

Comment:
This comment can be removed if the table does not include header and footer rows.